support-lambdas/acquisitions-firehose-transformer/src/test/scala/com/gu/acquisitionFirehoseTransformer/LambdaSpec.scala

```diff
@@ -88,6 +88,9 @@ class LambdaSpec extends AnyFlatSpec with Matchers {
       paymentId = None,
       queryParameters = Nil,
       platform = None,
+      postalCode = Some("postalCode"),
+      countrySubdivisionCode = Some("countrySubdivisionCode"),
```

Comment:
I would rather call this `state` as that's what it's known as in most of the rest of this codebase

Reply:
Also this class doesn't need to know about the new fields as it doesn't use them, so let's not add them in.